research/notes.org

```diff
@@ -8,67 +7,95 @@ subject are linked.
 :EXPORT_FILE_NAME: spoofax-org-export
 :END:
 #+LATEX_CLASS: article-shifted
-Spoofax is an environment that allows for the completely /declarative/
-definition of a programming language. The definition is done using
+Spoofax is a platform that allows for the completely /declarative/
```

Comment:
`a completely declarative`? I don't think `the` and `completely` go together well...